### PR TITLE
distribute.launch use poll to query subprocess

### PR DESCRIPTION
**Pull request by wangxicoding**
Use async poll to query whether subprocess is completed properly, if one of subprocess abort, the main process will sense it and terminate other child processes.

### CODE REVIEW

**Review by gongweibao — DISMISSED** [review]
LGTM

File: python/paddle/distributed/launch.py [hunk 1]
```diff
@@ -36,16 +36,25 @@
...
+import time
 import six
 import copy
 from argparse import ArgumentParser, REMAINDER
 import paddle.fluid as fluid
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+log_handler = logging.StreamHandler()
+format1 = logging.Formatter(
```
Comment (gongweibao) [inline python/paddle/distributed/launch.py]: Polish this name?

Comment (wangxicoding) [reply python/paddle/distributed/launch.py]: Done. change format1 to log_format

**Review by gongweibao — APPROVED** [review]
LGTM

**Pull request merged**